Makefile

```diff
@@ -74,12 +75,12 @@ shell: _requires-sandbox-up  ## Drop into a development shell
 .PHONY: register
 register: _requires-sandbox-up  ## Register Flyte cookbook workflows
 	$(call LOG,Registering example workflows in cookbook/$(EXAMPLES_MODULE))
-	$(call RUN_IN_SANDBOX,make -C cookbook/$(EXAMPLES_MODULE) register)
+	make -C cookbook/$(EXAMPLES_MODULE) register
```

Comment:
We will have to do either one of two things:
Option 1:
```suggestion
	$(call RUN_IN_SANDBOX,make -C cookbook/$(EXAMPLES_MODULE) serialize)
	make -C cookbook/$(EXAMPLES_MODULE) register
```
And make the `register` and `fast_register` targets in leaf.mk NOT run `serialize` and `fast_serialize` respectively.

Or Option 2:
```suggestion
	$(call RUN_IN_SANDBOX,make -C cookbook/$(EXAMPLES_MODULE) register)
```
And keep `register` and `fast_register` in leaf.mk depend on `serialize` and `fast_serialize` respectively.

Reply:
done